packages/notification/src/components/GenericNotification.spec.tsx

```diff
@@ -27,7 +27,7 @@ test('Begins "hidden" but transitions out of it immediately', async () => {
 
 test("The cancel button hides the notification and triggers the onHide callback", async () => {
   const onHide = jest.fn()
-  const { container, getByTitle } = render(
+  const { container, getByTestId } = render(
```

Comment:
Any idea why this broke, and if it will affect anything beyond these test cases?

(If the problem is with a change in react-testing-library, I guess we can assume its local to this repo. If the problem is with a change in behaviour to our component it might be more concerning)

Reply:
The fact that at this commit you haven't even changed any dependencies is particularly weird to me.

Reply:
I had a couple of flakey tests locally - that all always worked in CI :( This might have just been one that only failed locally, but would have worked in CI without my changes.

Reply:
I've noticed this failing across everyone's local environments, thanks for fixing :)